gtsam/linear/tests/testPowerMethod.cpp

```diff
@@ -42,23 +40,22 @@ TEST(PowerMethod, powerIteration) {
   // test power iteration, beta is set to 0
   Sparse A(6, 6);
   A.coeffRef(0, 0) = 6;
-  A.coeffRef(0, 0) = 5;
-  A.coeffRef(0, 0) = 4;
-  A.coeffRef(0, 0) = 3;
-  A.coeffRef(0, 0) = 2;
-  A.coeffRef(0, 0) = 1;
-  Vector initial = Vector6::Zero();
+  A.coeffRef(1, 1) = 5;
+  A.coeffRef(2, 2) = 4;
+  A.coeffRef(3, 3) = 3;
+  A.coeffRef(4, 4) = 2;
+  A.coeffRef(5, 5) = 1;
+  Vector initial = Vector6::Random();
```

Comment:
Same as above -- I think it would be better to make this a hard-coded vector

Reply:
Good suggestion!